recipe/meta.yaml

```diff
@@ -1,16 +1,16 @@
 {% set name = "fabio" %}
-{% set version = "0.14.0" %}
+{% set version = "2022.12.1" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 72ab0cb7247ec97dc552447112a54cd77b5fbe6f5ede8fc9b070e4e1c13cf01e
+  sha256: 668287dbb4c7c3042a404c80617bd6be9861a3ff50118a5a2b385f8a285f62bc
 
 build:
-  number: 2
+  number: 0
   script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv
```

Comment:
```
  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv
```

Can't you change this to something calling `meson` directly rather than going through `pip` to do the install to workaround https://github.com/mesonbuild/meson-python/issues/236?

Reply:
Good idea, let's give it a try